AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -185,6 +194,14 @@ - (void)clearFetchedData
   [super clearFetchedData];
     
   if ([self _shouldClearFetchedImageData]) {
+    
+    [_phImageRequestOperation cancel];
```

Comment:
Should this really be conditional inside _shouldClearFetchedImageData?  Given that it hasn't downloaded yet...a cancellation is a little different than clearing the fetched data, I'd say.